project/sites/communityrelations/themes/communityrelations_theme/src/scss/components.scss

```diff
@@ -35,6 +52,14 @@
     padding: 1rem 2.8rem;
     border-radius: 5px;
 
+    @media screen and (max-width: $tab) {
+      font-size: 1.6rem;
```

Comment:
You should use this @include adjust-font-size-to for all your font sizes.

Reply:
Done. thank you Adam